lib/src/coq.rs

```diff
@@ -168,6 +168,9 @@ pub(crate) enum Expression<'a> {
         /// a flag, set if implicit arguments are deactivated with '@'
         no_implicit: bool,
     },
+    Paren {
```

Comment:
```suggestion
    // TODO: remove to have parenthesis automatically detected
    Paren {
```
Adding a TODO comment